diff -u a/confirmation_instructions.html.erb b/confirmation_instructions.html.erb
--- a/confirmation_instructions.html.erb	2017-03-13 22:20:06.555536704 +0900
+++ b/confirmation_instructions.html.erb	2017-03-13 22:20:42.735004462 +0900
@@ -2,4 +2,4 @@
 
 <p>You can confirm your account email through the link below:</p>
 
-<p><%= link_to 'Confirm my account', confirmation_url(@resource, :confirmation_token => @resource.confirmation_token) %></p>
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, :confirmation_token => @token) %></p>
diff -u a/reset_password_instructions.html.erb b/reset_password_instructions.html.erb
--- a/reset_password_instructions.html.erb	2017-03-13 22:20:06.555536704 +0900
+++ b/reset_password_instructions.html.erb	2017-03-13 22:20:42.735004462 +0900
@@ -2,7 +2,7 @@
 
 <p>Someone has requested a link to change your password, and you can do this through the link below.</p>
 
-<p><%= link_to 'Change my password', edit_password_url(@resource, :reset_password_token => @resource.reset_password_token) %></p>
+<p><%= link_to 'Change my password', edit_password_url(@resource, :reset_password_token => @token) %></p>
 
 <p>If you didn't request this, please ignore this email.</p>
 <p>Your password won't change until you access the link above and create a new one.</p>
diff -u a/unlock_instructions.html.erb b/unlock_instructions.html.erb
--- a/unlock_instructions.html.erb	2017-03-13 22:20:06.555536704 +0900
+++ b/unlock_instructions.html.erb	2017-03-13 22:20:42.735004462 +0900
@@ -4,4 +4,4 @@
 
 <p>Click the link below to unlock your account:</p>
 
-<p><%= link_to 'Unlock my account', unlock_url(@resource, :unlock_token => @resource.unlock_token) %></p>
+<p><%= link_to 'Unlock my account', unlock_url(@resource, :unlock_token => @token) %></p>
